src/spec/ResourceSpec.ts

```diff
@@ -23,6 +23,23 @@ describe('A generic Resource', () => {
       expect(resource.isLoaded()).toBe(false);
       
    });
+
+   describe('without data', () => {
+      it('should not fail on load', (done) => {
+         var emptyLoader = new ex.Loader();
+         var game = new ex.Engine({
```

Comment:
This is the second place where this exact code is being used to create an engine--now might be a good time to create a TestUtils helper to create a testable engine (maybe default width/height to 500 but allow override).

```ts
var game = TestUtils.engine()
```